src/containers/Results/Results.js

```diff
@@ -5,52 +5,53 @@ import { connect } from 'react-redux'
 import { Link } from "react-router-dom";
 
 const Results = ({solarData, history}) => {
```

Comment:
I'll go into this file on the next PR and save so that it auto fixes the indentation in this file.